src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectMetaService.cs

```diff
@@ -63,6 +85,35 @@ public class SubjectMetaService : ISubjectMetaService
                     Value = item.Id.ToString()
                 });
         }
+        private static IEqualityComparer<Filter> FilterComparer { get; } = new FilterEqualityComparer();
+        private static IEqualityComparer<FilterGroup> FilterGroupComparer { get; } = new FilterGroupEqualityComparer();
+        protected Dictionary<string, TableBuilderFilterMetaViewModel> GetFilters2(IQueryable<Observation> observations)
```

Comment:
GetFilters2 can be called GetFilters now, but it's all the same code that is in AbstractTableBuilderSubjectMetaService, so extend that instead?